### PR TITLE
Add subscription info for removing azure stack registration

### PR DESCRIPTION
**Pull request by rakku-ms**
Bug 7264521: Update registration documentation with re-registration section (https://msazure.visualstudio.com/One/_workitems/edit/7264521)

### CODE REVIEW

**Comment by PRMerger8** [comment]
@rakku-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/operator/azure-stack-registration.md [hunk 1]
```diff
@@ -363,17 +363,25 @@ You need to update or renew your registration in the following circumstances:
 If you want to change the subscription you use, you must first run the **Remove-AzsRegistration** cmdlet, then ensure you're signed in to the correct Azure PowerShell context. Then run **Set-AzsRegistration** with any changed parameters including `<billing model>`. While running **Remove-AzsRegistration**, you must be signed in to the subscription used during the registration and use values of the `RegistrationName` and `ResourceGroupName` parameters as shown in the administrator portal [Find current registration details](#verify-azure-stack-hub-registration):
 
   ```powershell  
+  # select the subscription used during the registration
```
Comment (sarathys) [inline azure-stack/operator/azure-stack-registration.md:366]: you can locate this Azure Subscription from the AzureStack Admin Portal. (specify location)

Comment (rakku-ms) [reply azure-stack/operator/azure-stack-registration.md:366]: I have added the link to find the current registration details in the last sentence of line 363.

**Comment by PRMerger10** [comment]
@rakku-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@rakku-ms The PR is still in draft mode. The updates have been approved by the content team. Would you change the PR status to "Ready for review"? Then, we can merge the PR. Thanks!

image

**Comment by rakku-ms** [comment]
> @rakku-ms The PR is still in draft mode. The updates have been approved by the content team. Would you change the PR status to "Ready for review"? Then, we can merge the PR. Thanks!
> 
@ktoliver I am still working on few changes based on review comments. I will change the PR status to "Ready for review" by today EOD.

**Comment by PRMerger16** [comment]
@rakku-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@rakku-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by sethmanheim** [comment]
@rakku-ms Of course it would be much better if you used the private repo, then you'd get the benefit of link validation, grammar checking, and preview (staging) links. The public repo is better for external contributions.

**Comment by PRMerger20** [comment]
@rakku-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rakku-ms** [comment]
> @rakku-ms Of course it would be much better if you used the private repo, then you'd get the benefit of link validation, grammar checking, and preview (staging) links. The public repo is better for external contributions.

@sethmanheim sorry, I am not aware of any such private repo. Can you please make me a contributor on the same?

**Comment by sethmanheim** [comment]
@rakku-ms Sure, we can leave this as-is for now, and I will ping you offline.

**Comment by ktoliver** [comment]
@IngridAtMicrosoft I can't determine whether material changes were made in commits after you approved, so we better have you take another look and add `#sign-off` if it looks good to you. Thanks!

**Comment by IngridAtMicrosoft** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**